server/utils/bcrypt/index.ts

```diff
@@ -0,0 +1,7 @@
+import bcrypt from 'bcrypt';
+
+const hashPassword = async (password: string) => bcrypt.hash(password, 10);
+
+const comparePassword = async (password:string, hash:string) => bcrypt.compare(password, hash);
```

Comment:
It's a method that you can use directly in your controller, no need to separate them